interactors/src/main/java/com/thefuntasty/interactors/interactors/BaseMayber.kt

```diff
@@ -0,0 +1,20 @@
+package com.thefuntasty.interactors.interactors
+
+import io.reactivex.Maybe
+
+abstract class BaseMayber<ARGS, T> : BaseInteractor() {
+
+    protected abstract fun prepare(args: ARGS): Maybe<T>
+
+    fun create(args: ARGS): Maybe<T> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/abd629572327cbabcfd5483dec148fd3bc6b5ac2/interactors/src/main/java/com/thefuntasty/interactors/interactors/BaseMayber.kt#L9"></span>Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
  </tbody>
</table>